src/helpers/numbers.ts

```diff
@@ -50,7 +60,7 @@ const getNumberRegex = memoize(function getNumberRegex(locale: Locale) {
 export function isNumber(value: string | undefined, locale: Locale): boolean {
```

Comment:
`\n\n a/random/date` is still detected as a date, while `\n\n aRandomNumber` is detected as a string. We should be consistent